websites/D/Doordash/presence.ts

```diff
@@ -0,0 +1,101 @@
+const presence = new Presence({
+		clientId: "1001831947611426877",
+	}),
+	started = Date.now();
+
+presence.on("UpdateData", async () => {
+	const path: string = document.location.pathname,
+		presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: started,
+		};
+
+	// If they're using includes statements, it means the page is not a static page (like home, orders, etc.)
+
+	// Store page
+	if (path.includes("/store/") && !path.includes("/search/")) {
+		const name = document.querySelector<HTMLSpanElement>(
+				"h1.styles__TextElement-sc-1jrzcv7-0"
+			).textContent,
+			image = document.querySelector<HTMLImageElement>(
+				".menu-header-styles__LogoInner-sc-1kgi6s8-6 > picture:nth-child(1) > img:nth-child(5)"
+			),
+			hideFood = await presence.getSetting<boolean>("hideFood");
+
+		if (
+			document.querySelector<HTMLImageElement>(
+				".styles__ModalContainer-sc-1r4qbfh-0"
+			) === null
```

Comment:
You may need to format this again (`yarn format`)
```suggestion
			!document.querySelector<HTMLImageElement>(
				".styles__ModalContainer-sc-1r4qbfh-0"
			)
```